sound/soc/sof/intel/hda-dai.c

```diff
@@ -839,6 +839,50 @@ struct snd_soc_dai_driver skl_dai[] = {
 		.channels_max = 8,
 	},
 },
+{
+	.name = "SSP-HP Pin",
+	.playback = {
+		.channels_min = 1,
+		.channels_max = 8,
+	},
+	.capture = {
+		.channels_min = 1,
+		.channels_max = 8,
+	},
+},
+{
+	.name = "SSP-SPK Pin",
+	.playback = {
+		.channels_min = 1,
+		.channels_max = 8,
+	},
+	.capture = {
+		.channels_min = 1,
+		.channels_max = 8,
+	},
+},
+{
+	.name = "SSP-BT Pin",
+	.playback = {
+		.channels_min = 1,
+		.channels_max = 8,
+	},
+	.capture = {
+		.channels_min = 1,
+		.channels_max = 8,
+	},
+},
+{
+	.name = "SSP-AUX Pin",
+	.playback = {
+		.channels_min = 1,
+		.channels_max = 8,
+	},
+	.capture = {
+		.channels_min = 1,
+		.channels_max = 8,
+	},
+},
```

Comment:
I don't understand the concept of a virtual DAI if I am honest. I thought the goal was to abstract the name of the dailink in the machine driver. I am having a hard time with this, and this doesn't work moving forward where all SSPs will have to be power managed by the host.